### ISSUE TITLE
unbale to open more than one project (java/gradle)

### ISSUE THREAD

**Issue by gagara** [neoclide/coc.nvim#4188]
## Result from CocInfo

vim version: VIM - Vi IMproved 9.0 9000246
node version: v18.8.0
coc.nvim version: 0.0.82-b6e723ca 2022-09-13 11:25:46 +0800
coc.nvim directory: /home/vgagara/.vim/plugged/coc.nvim
term: dumb
platform: linux

## Describe the bug

Only one java project can be loaded with `coc-java` extension.
All following projects are not loaded into jdt workspace. `:CocCommand java.projectConfiguration.update` does nothing for them.

## Reproduce the bug

- Create file `mini.vim` with:
```vim
set nocompatible
set runtimepath^=/home/vgagara/.vim/plugged/coc.nvim
filetype plugin indent on
syntax on
set hidden
```
- Start (neo)vim with command: `vim -u mini.vim`
- Open first java project -> `jdt.ls` started and project is loaded as expected.
- Open another java project ->  nothing happened. Project is not added to jdt.ls workspace

## Additional observation

It looks like the issue was introduced by commit https://github.com/neoclide/coc.nvim/commit/a2f7d232d7d5e05c3aecd47074c49bce625b5a0c
downgrading to previous commit fixes the issue.

**Comment by chemzqm** [comment]
Not supported by coc-java yet.

**Comment by gagara** [comment]
what exactly not supported?
Opening several projects? In fact it is supported.
Note:
> downgrading to previous commit fixes the issue.

**Comment by chemzqm** [comment]
Should be fixed by 02ed2101
